docs/source/conf.py

```diff
@@ -2726,6 +2726,7 @@
     "SymExprPrinter",
     "RelaxedUnspecConstraint",
     "RuntimeAssert",
+    "ShapeGuardCppPrinter",
```

Comment:
All new public APIs need to be properly exposed and documented on the website.
For this, I guess you just want to properly make it private?